cluster/examples/kubernetes/ceph/csi/cephfs/snapshotclass.yaml

```diff
@@ -1,5 +1,13 @@
 ---
-apiVersion: snapshot.storage.k8s.io/v1beta1
+# Snapshot API version compatibility matrix:
+# v1betav1:
+#   v1.17 =< k8s < v1.20
+#   2.x =< snapshot-controller < v4.x
+# v1:
+#   k8s >= v1.20
+#   snapshot-controller >= v4.x
+# It is recommend to use {sidecar, controller, crds} of same version
+apiVersion: snapshot.storage.k8s.io/v1
```

Comment:
How about this example to help show the apiversion?
```suggestion
# 1.17 <= K8s <= v1.19
# apiVersion: snapshot.storage.k8s.io/v1beta
# K8s >= v1.20
apiVersion: snapshot.storage.k8s.io/v1
```

Reply:
yes, Its more helpful this way. I'll change it.